### ISSUE TITLE
Pre-winter 2025 Cleanup

### ISSUE THREAD

**Issue by nick-ulle** [ucdavisdatalab/adventures_in_data_science#13]
Update the reader to be more beginner-friendly and follow our guidelines in the [DataLab handbook](https://github.com/datalab-dev/handbook/blob/main/workshops/reader-style-guide.md).

Tasks:
- [ ] Check that vocabulary words are in **bold**
- [ ] Remove `$` and other prompt junk from command line examples (it confuses learners, differs between systems, and makes commands difficult to copy)

This list will grow as I spot problems while migrating the reader to Quarto.

**Comment by nick-ulle** [comment]
In the interest of time, I'm only including detailed notes for the core content (chapters 1-10).

**Comment by elisehellwig** [comment]
I think it might make sense to explicitly cover read.csv in the introduction to R (Basically what I did this year) since it seems like we want to have people read in data as part of the homework for that lecture. I know we are going to cover files, paths etc later, but since they have already seen paths with the command line, it doesn't seem like too heavy a lift. It is also the primary way people actually create data.frames.

@PLNReynolds also mentioned that it would make more sense to do at least some of the the reproducibility talk later on, once students have felt the struggle of running code a bit more.

**Comment by nick-ulle** [comment]
@elisehellwig I agree, I've always felt the sequence should be closer to R Basics.

I think we added in the bit about reproducibility with the intention to "motivate everything, " but it seems premature to me.

**Comment by nick-ulle** [comment]
I've replaced most of Chapter 4 with some new examples and some parts of R Basics and Intermediate R. It should now be more on-point (e.g., not teaching infrequently-used things like `repeat` and `switch`), but we should definitely do another pass later on to make sure the material is all accessible to our audience.

**Comment by nick-ulle** [comment]
We replaced most of the reader content in #26, so I'm closing this issue.
